src/sentry/search/snuba/executors.py

```diff
@@ -453,6 +465,25 @@ def trend_aggregation(start: datetime, end: datetime) -> Sequence[str]:
     ]
 
 
+def better_priority_aggregation(
+    start: datetime,
+    end: datetime,
+    aggregate_kwargs: Optional[PrioritySortWeights],
+) -> Sequence[str]:
+    event_age_hours = "divide(now() - timestamp, 3600)"
+    event_score = 1
+    min_score = 0.01
+    max_pow = 16
+    event_halflife_hours = 4  # halves score every 4 hours
+    aggregate_event_score = f"least({min_score}, sum(divide({event_score}, pow(2, greatest({max_pow}, divide({event_age_hours}, {event_halflife_hours}))))))"
```

Comment:
I think this should be `greatest`? Since we want to make sure the value never goes below `min_score`. `least` would cause us to select the `min_score` whenever the real score is higher.

Same for `greatest`, it should probably be `least` for the inverse reason.

We should probably have a test that would catch this.

Reply:
Oh oops I replaced min and max with them but that's the inverse 🙃